packages/provider/src/tests/accounts.ts

```diff
@@ -16,15 +16,16 @@ import { IDappAccount, IProviderAccount } from '@prosopo/types'
 import { Payee } from '@prosopo/captcha-contract/types-returns'
 import { ProviderEnvironment } from '@prosopo/types-env'
 import { Tasks } from '../index.js'
+import { TestAccount } from '@prosopo/env'
 import { getPairAsync } from '@prosopo/contract'
 
-export const accountMnemonic = (account: Account) => account[0]
-export const accountAddress = (account: Account) => account[1]
-export const accountContract = function (account: Account): string {
-    if (account[2]) {
-        return account[2]
+export const accountMnemonic = (account: TestAccount) => account.mnemonic
+export const accountAddress = (account: TestAccount) => account.address
+export const accountContract = function (account: TestAccount): string {
+    if (account.contractAddress) {
+        return account.contractAddress
     }
-    throw new Error(`Account ${account[1]} does not have a contract`)
+    throw new Error(`Account ${account} does not have a contract`)
```

Comment:
Are we happy to use Error instead of some sort of ProsopoError since this is in tests?

Reply:
it's just old code. I'll update.